### DISCUSSION TITLE
Unable to switch to the team plan on my organization

### DISCUSSION THREAD

**Discussion by CreaCloudz** [community/community#63553] [New to GitHub]
Hi,
I have an organization and I would like to pass it in Team plean I am currently in Free but I want to change I am on a pgae with the payment which turned in loop and crash I tested on three different computer and browser nothing changes .

**Comment by bf6** [comment]
+1, also happening to me

**Comment by queenofcorgis** [comment]
Hi there, 

It looks like a change our end caused this occur, we have since fixed it, but please let me know if you're still experiencing this situation! Thanks for reporting!
